tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -13,22 +13,25 @@ def __init__(self, *args, **kwargs):
     def run(self):
         if "Node[]" not in self.parameters:
             integration_id = self.parameters['TendrlContext.integration_id']
-
             _cluster = NS.tendrl.objects.Cluster(
                 integration_id=NS.tendrl_context.integration_id).load()
-            if (_cluster.import_job_id is not None and
-                    _cluster.import_job_id != "") or _cluster.import_status \
-                    in ["in_progress", "done", "failed"]:
+            if (_cluster.status is not None and
+                    _cluster.status != "" and
+                    _cluster.status in ["syncing", "importing", "unmanaging"]):
```

Comment:
Status field maintains status of any jobs running on the cluster. If import successful it would be set as syncing by GI. State of the cluster is maintained under final details as healthy /unhealthy. For afresh detected cluster status would be empty and I'd user submit import through app for a cluster which has import running back-end would reject the job.

Reply:
ack

Reply:
ok